app/assets/stylesheets/govuk_publishing_components/components/_attachment.scss

```diff
@@ -10,6 +10,11 @@ $thumbnail-icon-border-colour: govuk-colour("grey-3");
 .gem-c-attachment {
   @include govuk-font(19);
   @include govuk-clearfix;
+  position: relative;
+
+  .govuk-details__summary {
+    @include govuk-font($size: 14);
```

Comment:
Given @soniaturcotte's latest comments, do we want to use a different font here?

Reply:
No same font, just don't have to remove the arrow and grey border inset